powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/database/model/enumeration/ActivationOtpValidation.java

```diff
@@ -22,8 +22,8 @@
  * Enum representing possible activation OTP validation stages. Following values are supported:
  * <p>
  * - NONE = 0
- * - KEYS_EXCHANGE = 1
- * - COMMIT = 2
+ * - ON_KEY_EXCHANGE = 1
+ * - ON_COMMIT = 2
```

Comment:
🤔 In `CommitPhase.java`, there is `ON_COMMIT=0` https://github.com/wultra/powerauth-server/pull/1671/files?short_path=22681d7#diff-2e2152b1c4d5a479a3e8582131e414cc563391841b61655c12473a169495aa81R24
I realize, that it is a different enum, but at first sight, it is confusing.

Reply:
Yes, it is different.